File: Php/app/Urls/HomegameJoinConfirmationUrlModel.php
namespace app\Urls{

	use app\RouteFormats;
	use app\Urls\BaseClasses\HomegameUrlModel;
	use entities\Homegame;

	class HomegameJoinConfirmationUrlModel extends HomegameUrlModel{

		public function __construct(Homegame $homegame){
			parent::__construct(RouteFormats::homegameJoinConfirmation, $homegame);
		}

	}

}

File: Web/Models/UrlModels/HomegameJoinConfirmationUrlModel.cs
using Core.Classes;
using Web.Routing;

namespace Web.Models.UrlModels{

	public class HomegameJoinConfirmationUrlModel : HomegameUrlModel{

		public HomegameJoinConfirmationUrlModel(Homegame homegame)
			: base(RouteFormats.HomegameJoinConfirmation, homegame)
        {
		}

	}

}